propagation/propagation_test.go

```diff
@@ -25,7 +25,7 @@ import (
 type ctxKeyType uint
 
 var (
-	ctxKey ctxKeyType = 0
+	ctxKey ctxKeyType
```

Comment:
```suggestion
	ctxKey ctxKeyType = iota
```

Would it complain about this?  I'm not sure I like just leaving a bare variable declaration there even if I know it'll receive `0`.  I still have to spend some time thinking about it and why it's okay every time I look at the variable.

Reply:
The linter complains because `0` is already the zero value, thus initialising it with `0` is redundant. On the other hand. to use `iota` would change the meaning here. Even thought `ctxKey ` would have the value `0`, as it's the fist value used by `iota`, it'd indicate that more variables should be expected and for a context key, it isn't the case.